mappings/net/minecraft/recipe/RecipeType.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS azp net/minecraft/recipe/RecipeType
+	FIELD a CRAFTING Lazp;
+	FIELD b SMELTING Lazp;
+	FIELD c BLASTING Lazp;
+	FIELD d SMOKING Lazp;
+	FIELD e CAMPFIRE_COOKING Lazp;
+	METHOD a getIfMatches (Lazm;Lbci;Lafq;)Ljava/util/Optional;
```

Comment:
You sure that's a good name?

Reply:
It returns `Optional.of(recipe)` (`recipe` is the first parameter)  if the recipe matches, of `Optional.empty()` otherwise. Any suggestion for a different name?

Reply:
I wonder if `get` on its own is sufficient? The type should indicate that it only returns if it does match.

Reply:
You're right, that's better. I changed it to `get`.